content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -22,7 +22,7 @@ In Istio 1.4, we introduce an alpha feature to support {{< gloss >}}trust domain
 1. Install Istio with a custom trust domain and mutual TLS enabled.
 
     {{< text bash >}}
-    $ istioctl install --set profile=demo --set values.global.trustDomain=old-td
+    $ istioctl install --set profile=demo --set meshConfig.trustDomain=old-td
```

Comment:
There's another PR (#8301) that fixes this plus an IstioOperator example as well. It shouldn't be needed here, though, because the old helm API should also work for backward compatibility, right?

Reply:
Agree that his change isn't needed here. Leaving it the old ways, generates a warning message in the test output:
```
! values.global.trustDomain is deprecated; use meshConfig.trustDomain instead
```

Making this change just removes the warning. And since the value is deprecated, we are ready for the future.